src/cascadia/ut_app/SettingsTests.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+#include "../TerminalApp/ColorScheme.h"
+
+using namespace Microsoft::Console;
+using namespace TerminalApp;
+using namespace WEX::Logging;
+using namespace WEX::TestExecution;
+
+namespace TerminalAppUnitTests
+{
+    // Unfortunately, these tests _WILL NOT_ work in our CI, until we have a lab
+    // machine available that can run Windows version 18362. Until then, these
+    // tests will be commented out. GH#1012 should move our CI to that version.

```

Comment:
I finished 1012, but I don't think server 2019 is 18362 yet... you should double check though.

Reply:
Also, instead of leaving the tests commented out, you could wrap them with a function/helper/block that checks the OS version and returns the TAEF test-skipped status (much like the ones that I wrote that do that for OneCore editions). Then at least they'll run on some developer desktops and light up automatically when the test OS supports it.

Reply:
So after I dug into this, it turns out we can't preempt this test's deployment. The CI will try deploying the appx it runs in _before_ executing any test code. The CI is running `10.0.17763.557` currently. When it trys deploying the package, it'll fail, before we have the chance to say "please skip me", because the package itself is marked for 18295.
#wontfix